src/firefly/java/edu/caltech/ipac/firefly/server/util/VersionUtil.java

```diff
@@ -58,6 +59,7 @@ public static void initVersion(ServletContext context) {
         if (context!=null && !init) {
             init= true;
             ingestVersion(context);
+            ServerContext.init(context.getContextPath(), context.getServletContextName(), context.getRealPath(WEBAPP_CONFIG_LOC));
```

Comment:
When I say 'move this line here', I meant to move this line
```java
    Logger.info("", VersionUtil.getVersionInfo());
```
here.  Not ServerContext.init.

Reply:
My bad, it works perfectly now!